### PR TITLE
FLPATH-224: Add notification-service UI

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

**Comment by mareklibra** [comment]
/hold

Waiting on back-end fixes to finalize API calls

**Review by dagda1 — APPROVED** [review]
Nice PR!  good work

File: plugins/parodos/src/components/notification/NotificationList.tsx [hunk 1]
```diff
@@ -1,148 +1,255 @@
...
+          });
+        } catch (_) {
+          errorApi.post(
+            new Error(
+              `Failed to delete notification: ${JSON.stringify(notification)}`,
+            ),
+          );
+        }
+      };
+      doItAsync();
```
[the remark this answers is not in the document]
Comment (dagda1) [reply plugins/parodos/src/components/notification/NotificationList.tsx:111]: I mean in the other places where you have used `doItAsync`

**Review by RichardW98 — APPROVED** [review]

File: plugins/parodos/src/components/notification/NotificationList.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (mareklibra) [reply plugins/parodos/src/components/notification/NotificationList.tsx:111]: Done, not needed here.

[the remark this answers is not in the document]
Comment (mareklibra) [reply plugins/parodos/src/components/notification/NotificationList.tsx:111]: Actually the flow would be more complex without `await`, so I prefer to keep it.
The re-fetch needs to be called after the action is finisdhed.

[the remark this answers is not in the document]
Comment (mareklibra) [reply plugins/parodos/src/components/notification/NotificationList.tsx:111]: I moved the `async` one level up

**Review by RichardW98 — APPROVED** [review]

**Pull request merged**